src/components/Navigation/Navigation.tsx

```diff
@@ -30,7 +29,8 @@ export const Navigation: React.FunctionComponent<NavigationProps> & {
   onDismiss,
   ariaLabelledBy,
 }: NavigationProps) {
-  const {logo} = useTheme();
+  // TODO: This behavior will be re-enabled in a separate PR.
+  const logo = undefined as {[key: string]: any} | undefined;
```

Comment:
Drawing attention here to ensure this is capture in the following PR: https://github.com/Shopify/polaris-react/pull/4667